go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -851,6 +851,12 @@ func (tc *Catalog) FlushCollectionCompaction(ctx context.Context, flushCollectio
 		}
 		flushCollectionInfo.TenantLastCompactionTime = lastCompactionTime
 
+		// update collection total records
+		err = tc.metaDomain.CollectionDb(txCtx).UpdateTotalRecordsPostCompaction(flushCollectionCompaction.ID.String(), flushCollectionCompaction.TotalRecordsPostCompaction)
```

Comment:
can this be clubbed as part of `UpdateLogPositionAndVersion` above instead of another DB query? why update the same row of the same table twice within the same txn?